src/styles/components/_code.scss

```
@@ -149,7 +148,8 @@ pre[class*='language-'] {
 .token.attr-value,
 .language-css .token.string,
 .style .token.string,
-.token.variable {
+.token.variable,
+.token.inserted {
```

Let's add a comment of what `.token.inserted` does so it's easier for us to debug. There's a lot of old code in this repo that we're not sure what's needed or not. By providing comments like these, we can start to figure out what is important.

```suggestion
// Code additions (indicated by a leading '+') within a diff.
.token.inserted {
```

Do the same for `.token.deleted`:
```css
// Code removals (indicated by a leading '-') within a diff.
.token.deleted {
```

---

Added .token.deleted comment in this commit. https://github.com/ionic-team/ionic-docs/pull/4158/commits/9176e5b61e61fce36634152883d956176c24d9dc